ci/kokoro/windows/bazel/common.cfg

```diff
@@ -0,0 +1,20 @@
+# Format: //devtools/kokoro/config/proto/build.proto
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+build_file: "google-cloud-cpp-spanner/ci/kokoro/windows/bazel/build.bat"
+timeout_mins: 120
+
+gfile_resources: "/bigstore/cloud-cpp-integration-secrets/spanner-integration-tests-config.sh"
```

Comment:
FYI: I am not sure this is going to be very useful, it is a bash file after all.

Reply:
Good point, done